components/AnchorNavigation/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+import "../index.scss";
+
+<Meta
+  title="CSS/Navigation/Anchor Navigation"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Anchor Navigation" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <nav class="denhaag-anchor-navigation" aria-labelledby="secondary-navigation">
+      <ul id="secondary-navigation">
+        <li>
```

Comment:
I'm missing all the CSS selectors :) This way there are no modifiers possible.

Reply:
There is no reason to style the li's

Reply:
That was an old commend before u added className at all ;-)